src/test/ui/generic-associated-types/issue-86483.stderr

```diff
@@ -1,12 +1,10 @@
 error[E0311]: the parameter type `T` may not live long enough
   --> $DIR/issue-86483.rs:5:1
    |
-LL |   pub trait IceIce<T>
-   |   ^                - help: consider adding an explicit lifetime bound...: `T: 'a`
-   |  _|
-   | |
+LL | / pub trait IceIce<T>
 LL | | where
 LL | |     for<'a> T: 'a,
+   | |                  - help: consider adding an explicit lifetime bound...: `+ 'a`
 LL | | {
 ...  |
 LL | |
```

Comment:
The previous suggestions were already "wrong", but the new output is "wrong". I wonder if we should "just" detect these and remove the suggestion.